manimlib/mobject/svg/brace.py

```diff
@@ -9,7 +9,7 @@
 from manimlib.mobject.types.vectorized_mobject import VMobject
 from manimlib.utils.config_ops import digest_config
 from manimlib.utils.space_ops import get_norm
-
+import copy
```

Comment:
(As I stated elsewhere) Should probably be in a separate general cleanup PR (for organization purposes)

Reply:
Same as above: This change should not be in my PR, as it has nothing to do with my changes.

Reply:
@PhillipImhof I added a suggestion here as well